client/blocks/reader-post-card/style.scss

```
@@ -148,16 +148,27 @@
 	}
 
 	&.is-compact {
-		.reader-featured-image {
-			width: 200px !important;
+		.reader-post-card__post-content:not(.reader-post-card__no-excerpt) {
+			.reader-featured-image {
+				width: 200px !important;
 
-			@include breakpoint-deprecated( "<660px" ) {
-				width: auto !important;
-				height: 300px !important;
+				@include breakpoint-deprecated( "<660px" ) {
+					width: auto !important;
+					height: 300px !important;
+				}
+
+				@include breakpoint-deprecated( "<480px" ) {
+					height: 200px !important;
+				}
 			}
+		}
 
-			@include breakpoint-deprecated( "<480px" ) {
-				height: 200px !important;
+		.reader-post-card__no-excerpt.reader-post-card__post-content {
```

Nit: Can we swap this so the class that's common to both is first?

It might also be easier to read/understand if we use a single `.reader-post-card__post-content` parent selector and then add `&:not(.reader-post-card__no-excerpt)` and `&.reader-post-card__no-excerpt` as children.
```suggestion
		.reader-post-card__post-content.reader-post-card__no-excerpt {
```